src/main/java/com/nilsenlabs/flavormatrix/actions/DimensionList.kt

```diff
@@ -36,50 +46,106 @@ class DimensionList {
         return dimensions.firstOrNull { it.flavors.contains(flavorSelectable) }
     }
 
-    fun selectFrom(androidModules: List<GradleAndroidModel>) {
-        for (module in androidModules) {
-            for (selectedFlavor in module.selectedVariant.productFlavors) {
-                getFlavorByName(selectedFlavor)?.isSelected = true
+    /** Select currently selected flavors from reflection-based android models */
+    fun selectFrom(androidModels: List<Any?>) {
```

Comment:
any reason you use the term "models" instead of "modules" here? Misspelling?

Reply:
I see now that the source class is named  `com.android.tools.idea.gradle.project.model.GradleAndroidModel` so I guess it is a model of a module